tests/Feature/AuthTokenTest.php

```diff
@@ -71,7 +43,7 @@ public function test_createTokenWhenNoneExists_returnsNewToken()
                          ->get('/auth/create-token');
 
         $response->assertStatus(302);
-        $response->assertRedirect('auth/api-settings');
+        $response->assertRedirect('store/api-settings');
```

Comment:
again: can we use `route('store.api-settings')` here, instead?

Reply:
```suggestion
        $response->assertRedirect(route('store.api-settings'));
```